kolibri/plugins/learn/assets/src/views/commonLearnStrings.js

```diff
@@ -16,6 +16,19 @@ export const learnStrings = createTranslator('CommonLearnStrings', {
     message: 'Resume',
     context: 'Label for links that go to content that has been started and can be resumed',
   },
+  classesAndAssignmentsLabel: {
+    message: 'Classes and assignments',
+    context: 'Label for links that go to class or lesson content',
+  },
+  channelAndFoldersLabel: {
+    message: 'Channel and folders',
+    context: 'Label for links that go to the main channel or its subfolders',
+  },
+  filterAndSearchLabel: {
+    message: 'Filter and search',
```

Comment:
* `section` element must have the `role="region" ` to be recognized as a landmark
* ``aria-label``, not ``arialabel``

![Ubuntu18 (start)  Running  - Oracle VM VirtualBox_019](https://user-images.githubusercontent.com/1457929/186936676-db06bba9-602f-4007-9827-6cc880f20ae3.png)